.azure/pipelines/richnav.yml

```diff
@@ -0,0 +1,64 @@
+#
+# See https://docs.microsoft.com/en-us/vsts/pipelines/yaml-schema for details on this file.
+#
+
+# Configure which branches trigger builds
+trigger:
+  branches:
+    include:
+    - blazor-wasm
+    - master
+    - release/*
+    - internal/release/*
+
+variables:
+- name: _BuildArgs
+  value: '/p:SkipTestBuild=true'
+- name: Windows86LogArgs
+  value: -ExcludeCIBinaryLog
+
+stages:
+- stage: build
+  displayName: Build
+  jobs:
+  # Build Windows (x64/x86)
+  - template: jobs/default-build.yml
+    parameters:
+      codeSign: false
+      jobName: Windows_build
+      jobDisplayName: "Build: Windows x64/x86"
+      enableRichCodeNavigation: true
+      agentOs: Windows
+      steps:
+      - script: ./build.cmd
+                -ci
+                -all
+                -arch x64
+                /p:EnableRichCodeNavigation=true
+                $(_BuildArgs)
+        displayName: Build x64
+
+      # Build the x86 shared framework
+      # This is going to actually build x86 native assets.
+      - script: ./build.cmd
+                -ci
+                -noBuildRepoTasks
+                -arch x86
+                -all
+                -noBuildJava
+                -noBuildNative
+                /p:EnableRichCodeNavigation=true
+                $(_BuildArgs)
+                $(Windows86LogArgs)
+        displayName: Build x86
+
+      # Windows installers bundle both x86 and x64 assets
+      - script: ./build.cmd
+                -ci
+                -noBuildRepoTasks
+                -buildInstallers
+                -noBuildNative
+                /p:AssetManifestFileName=aspnetcore-win-x64-x86.xml
+                /p:EnableRichCodeNavigation=true
+                $(_BuildArgs)
```

Comment:
This step may not be worthwhile unless we can include C++ in the rich nav languages. The Windows installers build only includes 2 C# projects.